### PR TITLE
Fix GitHub auth & implement logout feature

### PR DESCRIPTION
**Pull request by nesaulov**
#13 & #25 

@jarosluv please review

### CODE REVIEW

File: lib/docdog/accounts/accounts.ex [hunk 1]
```diff
@@ -103,7 +103,7 @@ defmodule Docdog.Accounts do
   end
 
   def find_or_create(%Auth{} = auth) do
-    query = from(u in User, where: u.username == ^auth.info.nickname)
+    query = from(u in User, where: u.email == ^auth.info.email)
```
Comment (nesaulov) [inline lib/docdog/accounts/accounts.ex:106]: I am not quite sure about this: does Github API always provide an email in the payload? Or if user decides to make his email private we will get `nil` here?

Comment (jarosluv) [reply lib/docdog/accounts/accounts.ex:106]: We need to use `github_uid` here. Please, read my comment in issue #25.

File: config/test.exs [hunk 2]
```diff
@@ -15,6 +15,6 @@ config :docdog, Docdog.Repo,
   adapter: Ecto.Adapters.Postgres,
   username: System.get_env("DATABASE_USERNAME") || "postgres",
   password: System.get_env("DATABASE_PASSWORD") || "postgres",
-  database: "docdog_test",
+  database: System.get_env("DATABASE_NAME") || "docdog_test",
```
Comment (jarosluv) [inline config/test.exs]: We don't need this commit.

**Comment by nesaulov** [comment]
@jarosluv I am afraid, you will have to continue by yourself from here, I'm having a really busy week, I think I won't have time for this, sorry. After that I will have more time and will contribute, but right now - just no possibility. I have pushed changes I've made since last update, some things are fixed.

**Comment by jarosluv** [comment]
@nesaulov 

Really thanks for your work! I just merged only logout feature from your branch in PR #33.
And now I'll implement fix for sign in feature.

Please, come back as soon as you have free time. Your participation really helps!

**Pull request closed without merging**